### DISCUSSION TITLE
Jellyseerr with Jellyfin credentials

### DISCUSSION THREAD

**Discussion by andrwb33** [tteck/Proxmox#3293] [Q&A] [answered]
I am unable to login to jellyseerr with my Jellyfin credentials. Everything else seems to work fine. I'm thinking it's because the script installs the developer version. Anyone else experiencing this?

**Comment by HearthCore** [comment, marked as answer]
Make sure the machines can reach each other and use http://url:port - not a reverse proxy
When setup correctly you should be able to login with the JellySeer admin credentials and your JellyFin credentials.

**Comment by chan32167** [comment]
yes same problem here, cant login it was working for months. This past 2 weeks or so stopped working
![image](https://github.com/tteck/Proxmox/assets/10813272/851f47f3-59e0-4bf2-b1a3-38e5682c28b2)
